react/global.css

```diff
@@ -213,3 +213,57 @@ body.vtex-filter-popup-open {
     grid-template-columns: repeat(2, 1fr);
   }
 }
+
```

Comment:
Given a look in the css code, i think that some attributes can be replaced with design tokens, like the borders, colors and grid markers. This should improve the stylization of the component. What do you think?